### ISSUE TITLE
Encouraging All Types of People to Attend

### ISSUE THREAD

**Issue by brewcore** [TechLancaster/community#3]
What things should we consider to make everyone feel welcome in our community?  How can we make women feel more comfortable attending meetups?

**Comment by elyseewing** [comment]
I think that in addition to women, two of the other groups in the community that we need to put some effort toward attracting is 1) younger folks, and 2) less experienced folks.

"Younger" generally being defined as college and/or high school students who are involved in tech and looking to network, meet like-minded people, etc., but can't come to meetups like Pub Standards because of the 21+ nature. Even Tech Lancaster, while not at a bar, has the "we go to Quips afterward" deal which basically means that anyone under 21 who comes to the meetup can only spend until 8:00 with us, and they're probably not likely to continue making the trip (especially if they're coming from a surrounding town/area) for just 1-1.5 hours of presentations (even if said presentations/discussions are awesome). I think there are meetups right now (Front-End, Hacknight, etc.) that based on their location do open up this option and so we should capitalize on that as we reach out to the local colleges when the new semester starts, but many of the other meetups as they exist right now would have to undergo some pretty significant changes to handle this issue. Obviously not every meetup has to be for every person (and I think we'd all go pretty insane if we tried to make that the case), just as long as we remain cognizant of the ratio and content/focus of meetups that do and do not allow for participation from this group. This could also be something that we decide is important to address in the form of a new meetup(s) with a different focus, venue, etc.

The "less experienced" group is another big one that I think has been brought to light more significantly by the ladies meetup but is by no means exclusive to it, because these people could be male or female and all of the issues still very much apply in either case. Right now, someone just interested in learning more about anything tech-related and hoping to do so by coming out to one or more of the meetups has a lot working against them, especially if they're at all insecure about their inexperience and therefore are afraid to show up, ask questions, and risk "outing" themselves as a beginner. I was talking with Nathan about this last night and he had a fantastic idea that I think we should absolutely talk about at the next TL (the idea right now is specific to TL but could easily be branched out to other meetups particularly where presentations happen): **each presentation at TL, regardless of topic, would have a 2-slide (or ~2 minute, if no slides are being used) "let's all get on the same page" intro that defines the major concepts that are going to be covered in that talk**, down to a reasonable base level of knowledge. This way, there's no "raise your hand if you know x", which while well-intentioned and worked for us so far, could definitely be very exclusionary or intimidating to anyone in the room who is watching 75% or more of the other people in the room raise their hand and thinking "I am so out of my league". If instead there is a brief intro, presented in a simple format, then at least everyone in the room, regardless of knowledge level, is on relatively similar footing going into the content of the talk and anyone who is newer to that technology at least has some helpful context that starts them off feeling a little more comfortable and not so fish-out-of-water. Obviously even with this intro slide concept we're still going to have to assume SOME base knowledge (we can't start every talk explaining what a computer is), but I still think this would be an easily-implementable way to lessen the intimidation factor that might prevent some less experienced folks from coming back or coming out at all. I think a discussion next month on determining what this base context is (e.g., we're okay with "JavaScript is a modern dynamic web programming language that is frequently used in the browser" as an intro slide point because we've decided the base knowledge level is somewhere around knowing what programming languages are) would be helpful, as well as general feedback on the idea both on here and at the meetup too. The other idea that I had last night which is somewhat similar to this (and again is currently specific to TL but could be applied anywhere with presentations) is that the speaker(s) would be asked to throw together a short summary of their talk beforehand (which is a good idea anyway so we can put it up on the website, etc. for people to decide if the topic(s) interest them enough to attend the meetup) and also **provide a list of some resources that people who aren't as familiar with their topic might want to check out before the meetup to gain some background knowledge**. It might be as simple as a wiki article, a few blog posts, whatever. So, in the case of last night, someone totally unfamiliar with unit testing in JavaScript could have read a bit about both of those things (using links that Court felt were most relevant, clear, or helpful) before coming out, which makes them feel much more at ease with the subject matter and, again, starts their experience off in a more positive and less potentially-alienating way. Other ideas (both short-term/easy-add like this one as well as long-term like new, beginner or intro-focused meetups) for being more beginner-friendly are welcome!

For the women - as Erin mentioned in Slack today, the biggest points of feedback we've heard from the ladies meetup are their issues about venue/setting (bars and booze) and also childcare. We're taking some steps in the ladies meetup specifically to work on both of these (including setting up a floating second monthly meetup that will happen on Fridays or Saturdays to give an opportunity for those who can't make it out during the week because of childcare issues, etc.) but it's definitely an important discussion to have in the context of the general meetups too.

I think overall the most important thing is to remember that while every meetup can't be everything to everyone, the more issues we have out in the open and the more groups/people we're consciously trying to include (or, almost more importantly, being conscious **not** to **exclude**), the more of a chance that our existing meetups and the new meetups yet-to-be-created will be as open and welcoming as possible.

< / novel >

**Comment by chadharnish** [comment]
Also keep in mind other event engines including this one which reaches a broader audience.  If you want to promote any event, listing is free
http://www.visitlancasterpa.net/Lancaster-County/Event-Calendar/Website-and-Marketing/

**Comment by brisayswhaat** [comment]
AIGA Central PA is also opening up their calendar to local events. You can contact our web chair, Ryaan Beck, to get any events listed. I've been working with him a bit to get some of the Amish Tech calendar on there. His email is ryaan@centralpa.aiga.org

**Comment by chadharnish** [comment]
WITF has RSVPA too... kind of a clunker if I remember, but reaches Harrisburg etc.

**Comment by brewcore** [comment]
My concern as we get events added to places like these is making sure they stay up to date as information changes, so we don't end up with large amounts of duplicate and incorrect information. Hopefully they can use a feed from the community calendar https://goo.gl/RzNDjq.

**Comment by eringood02** [comment]
Branching off of Elyse's novel :) I've reached out to many of the colleges in the area about my meetup group; while mentioning a few others in the area, and being that the meetup culture is a fairly new thing in Lancaster (at least since I was a student) they were absolutely thrilled to hear of such a thing. I think it would be a great opportunity to include students as much as we can, for learning/mentoring and networking purposes. 

Also to her second point, I think a small introduction definitely heightens the appeal for those not necessarily knee-deep in the tech field but who are still interested in learning more. 

I think in the near future we could definitely organize a 'Super Meetup' if you will; Philly just had one, obviously of a much larger scale than we would - and I've heard whispers about it from multiple corners of the area, but getting everyone together as a community to meet and say 'hey, this is what we do' could be really beneficial to open some doors and network in one place. Check it: http://www.meetup.com/Technically-Philly/events/223444289/

**Comment by brisayswhaat** [comment]
Yeah we're trying to just pull the feed with a wordpress plugin, I forget which.

**Comment by tomswartz07** [comment]
> Younger" generally being defined as college and/or high school students who are involved in tech and looking to network, meet like-minded people, etc., but can't come to meetups like Pub Standards because of the 21+ nature.

You can throw a rock and hit both Thaddeus Stephens and McCaskey from the Science Factory: younger folks would probably really be interested in these topics- these two schools could be prime locations for getting info out.

**Comment by elyseewing** [comment]
I think our biggest post-high school draws are definitely Millersville (extra credit because there are about a million of us who are alum and still have faculty connections), F&M, Thaddeus Stevens, PCAD, etc. Penn Manor has had a huge technology push in their curriculum/activities in recent years so in terms of high school students that's probably going to right up their alley. McCaskey has less tech curriculum base but our ability to offer that opportunity to students there is great. We're not lacking in students to reach out to, we just need to be able to have as much as possible to keep offering them when we do. :)

**Comment by pmathur08** [comment]
Hi guys! Read through this thread. I am a new member and unfortunately, I have not attended a meeting yet, but I would love to help out with attracting F&M crowd. Quick intro - Senior @ F&M, studying Business and Computer Science, and co-founder and CEO of [SimplyStow](https://simplystow.com/) tech startup in the logistics space (newly venture backed). I can get the computer science club and the department, especially because I think we can learn from your front end experience (since we get none at F&M). 

I also have a partnership with Product Hunt for SimplyStow. If you guys are interested, we could do a combined SimplyStow + TechLancaster + Product Hunt event. Are you guys interested in that?

BTW super small plug - we are pre-launching on [Product Hunt tomorrow](https://www.producthunt.com/upcoming/simplystow) and I would love the support.

Let me know your thoughts.
